README.md

```diff
@@ -14,7 +14,7 @@
 ## Getting Started
 
 - Click the link below to get started with a live demo
-  [homepage.html](https://htmlpreview.github.io/?https://github.com/DebbyMiressa/MyPortfolio/blob/recent-work-and-about/homepage.html)
+  [homepage.html](https://htmlpreview.github.io/?https://github.com/DebbyMiressa/MyPortfolio/blob/add-contact-form/index.html)
```

Comment:
- [ ] [optional] @DebbyMiressa and @aceDavon it would be nice adding a link to your issues page so that its easier to add contributions to your project.